src/components/side_panel/chart/chart_with_axis/design_panel.ts

```diff
@@ -207,6 +208,11 @@ export class ChartWithAxisDesignPanel extends Component<Props, SpreadsheetChildE
       element.value = `${this.getTrendLineConfiguration()?.order ?? 2}`;
       return;
     }
+    const maxOrder = this.getMaxPolynomialDegree();
+    if (order > maxOrder) {
+      element.value = `${this.getTrendLineConfiguration()?.order ?? maxOrder}`;
+      return;
```

Comment:
Wait so we change the value of the input, but do not call updateTrendLineValue ? Won't we have a mismatch between the chart and the input ? (same for `if (order < 2)`)

Reply:
Oh yeah we try to use `this.getTrendLineConfiguration()?.order` (which I'm gressing is from the chart definition).

Still, shouldn't we set the max/min value instead of going back to the old value ? If I write degree 40 in my input i'd expect the polynomial go go to the max degree rather that goign back to 2. Or maybe it's just me ?